packages/labs/test/selectTests.tsx

```diff
@@ -5,14 +5,14 @@
  * and https://github.com/palantir/blueprint/blob/master/PATENTS
  */
 
-import { Classes, InputGroup, Popover } from "@blueprintjs/core";
+import { Classes, InputGroup } from "@blueprintjs/core";
 import { assert } from "chai";
 import * as classNames from "classnames";
 import { mount } from "enzyme";
 import * as React from "react";
 
 import { Film, TOP_100_FILMS } from "../examples/data";
-import { ISelectItemRendererProps, ISelectProps, Select } from "../src/index";
+import { ISelectItemRendererProps, ISelectProps, Popover2, Select } from "../src/index";
```

Comment:
nit: Import from the source file, not from `index`.

Reply:
That's how we do it in the other tests. Is index okay for tests?

Reply:
index is fine for tests, I guess, cuz it's how we've always done it. not a big deal.